specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -322,6 +322,142 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/billingInstructions": {
```

Comment:
[nit] Prefixing all the resource names with "billing" feels redundant, since the parent resource types and provider name already encapsulate this - i.e. this resource is a `Microsoft.Billing/billingAccounts/billingProfiles/billingInstructions` resource rather than a `Microsoft.Billing/accounts/profiles/instructions` resource, which is more consistent with other providers.

Reply:
Agreed. Renamed it to just "instructions".